backend/migrations/20241216163427_first-kemomimi.up.sql

```diff
@@ -0,0 +1,42 @@
+-- 製品テーブル（product）作成
+CREATE TABLE
+    product (
+        product_id TEXT PRIMARY KEY, -- 製品のユニークID
+        name TEXT NOT NULL, -- 製品名
+        model_number TEXT, -- 型番
+        product_url TEXT, -- 商品URL
+        remarks TEXT -- 備考
+    );
+
+-- 備品テーブル（public_item）作成
+CREATE TABLE
+    public_item (
+        public_item_id TEXT PRIMARY KEY, -- 備品のユニークID
+        name TEXT NOT NULL, -- 備品名
+        product_id TEXT, -- 製品のユニークID（Foreign Key）
```

Comment:
```suggestion
        FOREIGN KEY product_id TEXT, -- 製品のユニークID（Foreign Key）
```
だったりしますか?

Reply:
テーブル成約として書いてあります

書くならこうですかね (`(product_id)` を省略することもできますが)

```suggestion
        product_id TEXT REFERENCES product (product_id), -- 製品のユニークID（Foreign Key）
```

Reply:
あっすいません。みてなかったです。ありがとう。

Reply:
これ記法改めたほうがよさそうですかね

Reply:
いや自分がよく見てなかっただけなので。
まあ鮭さん次第では